lib/procore-sift.rb

```diff
@@ -66,8 +66,8 @@ def sort_fields
   end
 
   class_methods do
-    def filter_on(parameter, type:, internal_name: parameter, default: nil, validate: nil, scope_params: [])
-      filters << Filter.new(parameter, type, internal_name, default, validate, scope_params)
+    def filter_on(parameter, type:, internal_name: parameter, default: nil, validate: nil, scope_params: [], tap: nil)
+      filters << Filter.new(parameter, type, internal_name, default, validate, scope_params, tap)
```

Comment:
there is a small note in the readme which says scope_params should be the last part of the definition but this seems ok to me.

Reply:
I went ahead and removed that note after adding some tests to make sure that ordering doesn't seem to matter.